cluster/gce/gci/configure-helper.sh

```diff
@@ -1937,6 +1947,15 @@ function start-kube-controller-manager {
   sed -i -e "s@{{flexvolume_hostpath}}@${FLEXVOLUME_HOSTPATH_VOLUME}@g" "${src_file}"
   sed -i -e "s@{{cpurequest}}@${KUBE_CONTROLLER_MANAGER_CPU_REQUEST}@g" "${src_file}"
 
+  if [[ -n "${KUBE_CONTROLLER_MANAGER_RUNASUSER:-}" && -n "${KUBE_CONTROLLER_MANAGER_RUNASGROUP:-}" ]]; then
+    run-kube-controller-manager-as-non-root
+    sed -i -e "s@{{runAsUser}}@${KUBE_CONTROLLER_MANAGER_RUNASUSER}@g" "${src_file}"
+    sed -i -e "s@{{runAsGroup}}@${KUBE_CONTROLLER_MANAGER_RUNASGROUP}@g" "${src_file}"
+  else
+    sed -i -e "s@{{runAsUser}}@0@g" "${src_file}"
+    sed -i -e "s@{{runAsGroup}}@0@g" "${src_file}"
```

Comment:
Is defaulting to root the existing behavior, or should we remove the pod security context when KUBE_CONTROLLER_MANAGER_RUNASUSER is not specified? 

Also, does the new container security context (especially "drop all capabilities") affects current cluster creation?

Reply:
Yes, defaulting to root is the existing behavior. Every pod runs as root unless otherwise specified. 

Is there anything specific regarding cluster creation that you are concerned about? I have tested this change by creating new clusters in GKE.